pkg/storage/pod_test.go

```diff
@@ -46,14 +46,23 @@ var _ = Describe("Pod storage", func() {
 
 		By("waiting for second batch before serving metrics")
 		Expect(s.Ready()).NotTo(BeTrue())
-		checkPodResponseEmpty(s, podRef)
+		ts, ms, err := s.GetPodMetrics(podRef)
```

Comment:
This is incorrect behavior. Fact that metric was not stored before doesn't mean that it has recently restarted.